python/cugraph-dgl/cugraph_dgl/utils/cugraph_storage_utils.py

```diff
@@ -11,23 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import numpy as np
+from cugraph.gnn.dgl_extensions.utils.sampling import eid_n, src_n, dst_n
```

Comment:
It will be as soon as we add CI to cugraph-dgl. I plan to move all of code to cugraph-dgl now especially now that we dont have to support 4 classes but only need 1 class in `cugraph-dgl`.